.github/workflows/redis-rs.yml

```diff
@@ -65,8 +65,18 @@ jobs:
               working-directory: ./glide-core/redis-rs/redis
 
             - name: Test
+              run: 
+              # TODO remove the concurrency limit after we fix test flakyness.
               run: |
-                  cargo test --release -- --test-threads=1 | tee ../test-results.xml
+                  versions_to_compare=$(printf "%s\n" "${{ matrix.engine.version }}" "7.1" | sort -V)
+                # Sort versions and get the latest version between the current version and 7.1, if the latest version is 7.1, skip the test.
+                  lt_version=$(echo "$versions_to_comare" | tail -n1)
+                  if [[ "7.1" == "$lt_version" ]]; then
+                      cargo test --release -- --test-threads=1 | tee ../test-results.xml
+                  else
+                      cargo test --release --features valkey-gte-7-2 -- --test-threads=1 | tee ../test-results.xml
+                  fi
+                      
```

Comment:
- name: Test
              # TODO remove the concurrency limit after we fix test flakyness.
              run: |
                  versions_to_compare=$(printf "%s\n" "${{ matrix.engine.version }}" "7.1" | sort -V)
                  # Sort versions and get the latest version between the current version and 7.1, if the latest version is 7.1, skip the test.
                  lt_version=$(echo "$versions_to_comare" | tail -n1)
                  if [[ "7.1" == "$lt_version" ]]; then
                      cargo test --release -- --test-threads=1 | tee ../test-results.xml
                  else
                      cargo test --release --features valkey-gte-7-2 -- --test-threads=1 | tee ../test-results.xml
                  fi